src/ext/integrations/test_integration.h

```diff
@@ -0,0 +1,20 @@
+#ifndef DD_INTEGRATIONS_TEST_H
+#define DD_INTEGRATIONS_TEST_H
+#include <stdlib.h>
+
+#include "configuration.h"
+#include "defered.h"
+
+static inline void _dd_test_initialize_defered_integration() {
+    char *test_defered = getenv("_TEST_DEFERED");
+    if (!test_defered) {
+        return;
+    }
+
+    static struct ddtrace_defered_integration _integrations[] = {
+        DDTRACE_DEFERED_INTEGRATION_LOADER("test", "public_static_method", "load_test_integration"),
```

Comment:
I might use some clarification here:
- `test`: the class that triggers the loading?
- `public_static_method`: the method/function name?
- `load_test_integration`: the script name?

Reply:
I added some comments describing how these macros work